snyk-operator-certified/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2019 Snyk Ltd.
```

Comment:
Should this be 2021?

Reply:
Asked Gareth the same a while ago and he said no - keep in mind it's based on https://github.com/snyk/kubernetes-monitor/blob/staging/LICENSE